package.json

```diff
@@ -7,7 +7,7 @@
   "dependencies": {
     "@mdi/font": "^4.9.95",
     "flickity": "^2.2.1",
-    "gatsby": "^2.24.86",
+    "gatsby": "^2.32.13",
```

Comment:
What was this update for?

Reply:
Not sure, but when I downloaded Gatsby, I think it defaulted to the 2.32.13 version